### PR TITLE
Disable interactions with sld in drawing mode

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

**Review by jamal-khey — COMMENTED** [review]
because you have moved isDrawingMode to redux. you can symplify the code by removing this props https://github.com/gridsuite/gridstudy-app/pull/2169/files#diff-0573c2bf57349fc35baec5d04d4b9d8ceb91603978f2ebe22758e40fae7b3cc7R348
and use the value from the store

**Review by jamal-khey — COMMENTED** [review]
we can still open SLD using search, may be see with the PO if we disable sld opening via search ?

**Review by souissimai — APPROVED** [review]
maybe should check if need also to avoid search  when drawing mode is enabled, but not really included in this US

File: src/components/map-viewer.jsx [hunk 1]
```diff
@@ -95,6 +95,8 @@ const MapViewer = ({
     const lineParallelPath = useSelector(
         (state) => state[PARAM_LINE_PARALLEL_PATH]
     );
+    const isDrawingMode = useSelector((state) => state.isDrawingMode);
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/map-viewer.jsx:98]: Done.

**Comment by AAJELLAL** [comment]
> maybe should check if need also to avoid search when drawing mode is enabled, but not really included in this US

I mentioned the point during the daily, the search can be used to zoom in on a piece of equipment.

**Comment by AAJELLAL** [comment]
> we can still open SLD using search, may be see with the PO if we disable sld opening via search ?

I mentioned the point during the daily, the search can be used to zoom in on a piece of equipment and it's not really an interaction with the map.

**Review by souissimai — APPROVED** [review]
tested, code OK

**Review by jamal-khey — APPROVED** [review]
Test OK
Code Ok

**Pull request merged**